index.html

```diff
@@ -275,12 +275,12 @@ <h3 id="control" data-export="" data-dfn-type="dfn">
   <p>
 We recognize that web technologies can be used
 to manipulate and deceive people, complicate isolation, and encourage addictive behaviors.
-We seek to mitigate against these potential abuses
+We mitigate these potential abuses
```

Comment:
Again.  (Keep in mind, I'm not saying we shouldn't try - quite the opposite -but if you make this as a "we will eradicate the ability for web technologies to be used to manipulate and deceive people", that seems like an impossible task.

Reply:
This document claims that it wants to reinforce a human-rights-centric approach to the web. The UDHR doesn't say "we'll try to prevent slavery and servitude" or "we will aim to give people the right to life, liberty and security" or "we will consider ways to avoid subjecting people to torture." None of the UDHR's aims are possible today, yet every time they are not met we have failed.

Every hedge encourages status quo bias and invites complacency.

For this category of document, I think we can only go two broad ways. Either they are aspirational, define ethics, endgame, etc. in which case there is no value in watering them down with hedging and vague aspirations to aim. We don't know what might _become possible_. Just yesterday I was reading about how a $24bn investment could produce mRNA vaccines for all 26 viral _families_ known to cause human illness. Huge if true — but it shouldn't change medical ethics. Or, conversely, these are governance documents, producing specific rules, supporting specific interpretations and actions, and those must be implementable and therefore realistic. Similarly, a universal viral vaccine would change a lot to medical protocols and the overall governance of medicine as deployed.

Reply:
Yes, but that's precisely my point - you can be aspiration and define ethics and endgame, or you can be pragmatic, implementable and realistic.  The UHDR is the former, _**and it is introduced as such**_, and its wording allows it.

> The General Assembly ... [p]roclaims this Universal Declaration of Human Rights as a common standard of achievement for all peoples and all nations, to the end that every individual and every organ of society, keeping this Declaration constantly in mind, shall strive by teaching and education to promote respect for these rights and freedoms and by progressive measures, national and international, to secure their universal and effective recognition and observance, both among the peoples of Member States themselves and among the peoples of territories under their jurisdiction. 
[articles follow]

I'm +1 for being aspirational.  But then the precise wording is important; you can't say "slavery does not exist".  You can say "we believe everyone should believe these things and work to these ends... [Article 4] No one shall be held in slavery".  (Which is essentially what the UHDR does.)

In this case, it is especially important, because your claims are not, in many cases, pragmatically POSSIBLE.  You cannot mitigate _any harm_ a feature might offer, or any possibility of the web being used to spread misinformation.

Reply:
Agreeing with Chris here - the ability of building block technology (such as that specified by standards) to effect eg human rights is often limited, and we should not create expectations that we can mitigate or avoid all harms.